utils/vale/.vale.ini

```diff
@@ -49,4 +49,7 @@ TokenIgnores = ({{[%<] .* [%>]}}.*?{{[%<] ?/.* [%>]}}), \
 # Exclude `{{<  myshortcode `This is some <b>HTML</b>, ... >}}`
 BlockIgnores = (?sm)^({{[%<] [^{]*? [%>]}})\n$, \
 (?s) *({{<\s?hint [^>]* ?>}}.*?{{< ?/ ?hint >}}), \
-(?s) *({{<\s?ref [^>]* ?>}})
\ No newline at end of file
+(?s) *({{<\s?ref [^>]* ?>}})
+
+# alex settings
+alex.profanitySureness = 1
```

Comment:
@plumbis I can't seem to get the `alex` styles to only report on profanity for level 1+ certainty. Do you have some ideas here? I'd prefer not to manually edit the `alex` style files if possible.

Reply:
We're not running [alex ](https://github.com/get-alex/alex) but using their definitions in Vale. Each alex level is a different file in the [/utils/vale/styles/alex](https://github.com/upbound/docs/tree/main/utils/vale/styles/alex) directory.

I say go ahead and edit the alex/ProfanityUnlikely file and remove `execution`.